src/Builder/CliMenuBuilder.php

```diff
@@ -568,6 +592,14 @@ private function propagateStyles(CliMenu $menu, array $items = [])
                     $subMenu->setStyle(clone $menu->getStyle());
                 }
 
+                if (!$subMenu->getCheckboxStyle()->hasChangedFromDefaults()) {
```

Comment:
Each submenu object gets a copy of radio and checkbox style object to keep. This is what is handed out to children items.